### PR TITLE
Different views for sheep and shepherds

### PR DESCRIPTION
**Pull request by stevenshuhyow**

### CODE REVIEW

File: server/api/event/index.js [hunk 1]
```diff
@@ -53,6 +50,26 @@ router.get('/user/:userId', function(req, res) {
   });
 });
 
+
+// Reurn a list of all events for a user where the user is a shepherd
```
Comment (ekong2) [inline server/api/event/index.js]: take this off

File: server/api/event/index.js [hunk 2]
```diff
@@ -53,6 +39,24 @@ router.get('/user/:userId', function(req, res) {
   });
 });
 
+
+// Reurn a list of all events that have not ended yet, filtering is done in EventStore
+router.get('/sheep/:userId', function(req, res) {
```
Comment (kevhuang) [inline server/api/event/index.js]: What is the purpose of :userId?

File: client/public/js/components/EventList.js [hunk 3]
```diff
@@ -30,7 +27,8 @@ var EventList = React.createClass({
   getInitialState: function() {
     // Set initial state to the initial set of events defined in EventStore
     return {
-      events: getEvents(),
+      shepherdEvents: EventStore.getAllEventsByShepherd(cookie.load('id')),
+      notShepherdEvents: EventStore.getAllEventsNotByShepherd(cookie.load('id')),
```
Comment (kevhuang) [inline client/public/js/components/EventList.js]: `getAllEventsByShepherd` and `getAllEventsNotByShepherd` on the EventStore do not take any arguments.

File: client/public/js/utils/EventAPI.js [hunk 4]
```diff
@@ -6,21 +6,40 @@ var EventAPI = {
...
+  //     // On success, return event data
+  //     return data;
+  //   }, function() {
+  //     // On error, return empty array
+  //     return [];
+  //   });
+  // },
+
+  /**
+   * Get all events for a shepherd
```
Comment (kevhuang) [inline client/public/js/utils/EventAPI.js]: Fix the description

File: client/public/js/utils/EventAPI.js [hunk 5]
```diff
@@ -6,21 +6,40 @@ var EventAPI = {
...
+   * @param {number} shepherdId The primary key of the user in the database
```
Comment (kevhuang) [inline client/public/js/utils/EventAPI.js]: There are no function parameters

**Comment by kevhuang** [comment]
:+1:

**Comment by dmsakamoto** [comment]
:+1:

**Pull request merged**